entrypoint-prod.sh

```diff
@@ -0,0 +1 @@
+poetry run gunicorn --config gunicorn_config.py --bind 0.0.0.0:${PORT:-5000} 'todo_app.app:create_app()'
```

Comment:
It would be good to add some shebangs (e.g. `#!/bin/bash`) to these files to specify what interpreter should be being used

Reply:
added this to line 1 of both entrypoint files